src/magellan_core/msg/Waypoint.msg

```diff
@@ -0,0 +1,2 @@
+geometry_msgs/Pose goal
```

Comment:
My case for using `geometry_msgs/Point` is just that it's a smaller message, and I'd expect the orientation will be zero'd/ignored for now.

Reply:
Yeah but there might be a time we want an orientation (i.e. hitting a cone from a certain side)

Reply:
Yeah, especially since if we wanted extra fields, it provides extra flexibility.